Doc/whatsnew/3.13.rst

```diff
@@ -122,11 +122,21 @@ Optimizations
 Deprecated
 ==========
 
+Pending Removal in Python 3.15
+------------------------------
+
 * :mod:`wave`: Deprecate the ``getmark()``, ``setmark()`` and ``getmarkers()``
   methods of the :class:`wave.Wave_read` and :class:`wave.Wave_write` classes.
   They will be removed in Python 3.15.
   (Contributed by Victor Stinner in :gh:`105096`.)
 
+Pending Removal in Python 3.16
+------------------------------
```

Comment:
I don't think this sections are needed.

Reply:
We have these for other releases:

* https://docs.python.org/3.13/whatsnew/3.11.html#pending-removal-in-python-3-12
* https://docs.python.org/3.13/whatsnew/3.12.html#pending-removal-in-python-3-13
* https://docs.python.org/3.13/whatsnew/3.12.html#pending-removal-in-python-3-14
* https://docs.python.org/3.13/whatsnew/3.12.html#pending-removal-in-future-versions

They help users plan when to update their code, and also help us later when the time comes to remove old code.

Reply:
OK. I have seen only 3.11.html. It only has "Pending removal in Python 3.12" (3.11+1) section and only "have been deprecated in earlier Python releases" are listed.

But now I see 3.12.html and you are right.